### DISCUSSION TITLE
How do you do exception Handling for AppSyncResolver

### DISCUSSION THREAD

**Discussion by adithep** [aws-powertools/powertools-lambda-python#2170] [Q&A] [answered]
For HTTP resolver, there are few ways to handle exception: https://awslabs.github.io/aws-lambda-powertools-python/2.14.1/core/event_handler/api_gateway/#exception-handling

But for `AppSyncResolver`, I have no idea who to handle exceptions.

**Comment by leandrodamascena** [comment]
Hi @adithep, indeed we don't have a way to do this easily. I think we need to create an issue to put this in our backlog and address this issue.

cc @heitorlessa @rubenfonseca

**Comment by rubenfonseca** [comment, marked as answer]
Thanks! I've converted this Discussion into a Feature Request. You can track it here https://github.com/awslabs/aws-lambda-powertools-python/issues/2184
